core_api/src/routes/file.py

```diff
@@ -115,11 +115,14 @@ def get_file(file_uuid: UUID, _user_uuid: Annotated[UUID, Depends(get_user_uuid)
     Returns:
         File: The file
     """
-    return storage_handler.read_item(file_uuid, model_type="File")
+    file = storage_handler.read_item(file_uuid, model_type="File")
+    if file.creator_user_uuid != user_uuid:
+        raise HTTPException(status_code=404)
```

Comment:
yes this file does exist but we dont want the requestor to know that as it isnt thiers